test/unit/org/apache/cassandra/tools/ToolRunner.java

```diff
@@ -389,6 +389,19 @@ public Exception getException()
         {
             return e;
         }
+
+        /**
+         * Checks if the stdErr is empty after removing any potential JVM env info output and other noise
+         *
+         * Some JVM configs may output env info on stdErr. We need to remove those to see what was the tool's actual
+         * stdErr
+         *
+         * @return The ToolRunner instance
```

Comment:
Nit: it doesn't return anything. Same for the other overload of `assertCleanStdErr`, which was already wrong before the changes.

Reply:
ill delete; copy/paste =D